Source/Scene/Points3DTileContent.js

```diff
@@ -367,6 +447,10 @@ define([
             }
         }
 
+        if (hasBatchIds) {
+            vs += 'attribute float a_batchId; \n';
```

Comment:
Did you verify this, for example, by temporarily tweaking the shader to map this to a color ramp?